interfaces/IContentDecryption.h

```diff
@@ -34,6 +34,25 @@ namespace Exchange {
         virtual RPC::IStringIterator* Systems() const = 0;
         virtual RPC::IStringIterator* Designators(const string& keySystem) const = 0;
         virtual RPC::IStringIterator* Sessions(const string& keySystem) const = 0;
+
+        enum Status : uint8_t {
+            BUSY,
+            SUCCESS,
+            FAILED
+        };
+
+        /* @event */
+        struct EXTERNAL INotification : virtual public Core::IUnknown {
+
+            enum {ID = ID_CONTENTDECRYPTION_NOTIFICATION};
+
+            /* @brief initialization status. */
+            virtual void initializationStatus(const std::string& drm,
+                                              const Status status) = 0;
+        };
+
+        virtual uint32_t Register(IContentDecryption::INotification* notification) { return Core::ERROR_NONE; };
```

Comment:
There are multiple options to be backwards compatible, one is to just create a new interface and QI for this new interface if it is implemented. SInce this is a minor change it is oke to "add" a default implementation. However, it is than manadatory to retrun an Error code if it is being called but there is not a real implementation so suggest to return an Error, like Core::ERROR_NOT_SUPPORTED in stead of not warning the callee..

Reply:
Done